blocked-edges/4.13.46-AzureDependencyLoopNotReady.yaml

```diff
@@ -0,0 +1,13 @@
+to: 4.13.46
+from: ^4[.](12[.].*|13[.].*)[+].*$
+url: https://issues.redhat.com/browse/MCO-1257
+name: AzureDependencyLoopNotReady
+message: |-
+  On Azure clusters, updated nodes will go to the NotReady state.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      group(cluster_infrastructure_provider{_id="",type="Azure"})
+      or
+      0 * group(cluster_infrastructure_provider{_id=""})
```

Comment:
All Azure clusters, including ARO clusters, are affected. Is this PromQL sufficient? I have noticed we can use the `cluster_operator_conditions{name="aro"}` metric to select just the ARO clusters. However, I am under the impression that ARO and Azure clusters will have their cluster infrastructure provider set to Azure. However, I am not 100% sure at the moment.